packages/cli/src/bundle/overrides.ts

```diff
@@ -3,7 +3,7 @@ import type { BundleArgs } from "@rnx-kit/metro-service";
 import { pickValues } from "@rnx-kit/tools-language/properties";
 import type { CliPlatformBundleConfig } from "./types";
 
-export type BundleConfigOverrides = {
+type BundleConfigOverrides = {
```

Comment:
```suggestion
type BundleConfigOverrides = { Pick<CLICommonBundleOptions, "entryFile" | "bundleOutput" ... >};
```

This should be based on the names/types found in `CLICommonBundleOptions`.